content/10.diagnostics.md

```diff
@@ -137,20 +137,37 @@ The assay had 95% positive predictive agreement and 100% negative predictive agr
-Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/&mu;l in 40-60 minutes [@doi:10.1101/2020.04.10.023358].
-It was also reported that electric field gradients can be used to control and accelerate CRISPR assays by co-focusing Cas12-gRNA, reporters, and target [@doi:10.1101/2020.05.21.109637].
+This new assay, known as All-In-One Dual CRISPR-Cas12a (AIOD-CRISPR), detected 4.6 copies of SARS-CoV-2 RNA per &mu;l of input in 40 minutes [@doi:10.1038/s41467-020-18575-6].
+Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/&mu;l in 40-60 minutes [@doi:10.1038/s41421-020-0174-y].
+It was also reported that electric field gradients can be used to control and accelerate CRISPR assays by co-focusing Cas12-gRNA, reporters, and target [@doi:10.1073/pnas.2010254117].
 The authors generated an appropriate electric field gradient using a selective ionic focusing technique known as isotachophoresis (ITP) implemented on a microfluidic chip.
 They also used ITP for automated purification of target RNA from raw nasopharyngeal swab samples.
 Combining this ITP purification with loop-mediated isothermal amplification, their ITP-enhanced assay to achieved detection of SARS-CoV-2 RNA (from raw sample to result) in 30 minutes.
 
+All these methods required upstream nucleic acid amplification prior to CRISPR-based detection. 
+They are relying on type V (Cas12-based) and type IV (Cas13-based) CRISPR systems.
+In contrast, type III CRISPR systems have the unique property of initiating a signalling cascade, which could boost the sensitivity of direct RNA detection. 
+A study tested this hypothesis using the type III-A CRISPR RNA (crRNA)-guided surveillance complex from Thermus thermophilus [@doi:10.1101/2020.10.14.20212670]. 
+They showed that activation of the Cas10 polymerase generates three products (cyclic nucleotides, protons, pyrophosphates) that can all be used to detect SARS-CoV-2 RNA. 
+Direct detection was 100-fold more sensitive than direct detection by Cas13. 
+Detection of viral RNA in patient samples still required an initial nucleic acid amplification step, but improvements may in the future remove that requirement.
+
+This goal of amplification-free detection was later achieved for a Cas13a-based system [@doi:10.1016/j.cell.2020.12.001]. 
+This approach combined multiple CRISPR RNAs to increase Cas13a activation, which is detected by a fluorescent reporter. 
+Importantly, because the viral RNA is detected directly, the test yields a quantitative measurement rather than a binary result. 
+The study also shows that fluorescence can be measured in a custom-made dark box with a mobile phone camera and a low-cost laser illumination and collection optics. 
```

Comment:
I would leave that part out if the accuracy mentioned below was not achieved by the phone camera approach because that could be misleading.

Reply:
These results were achieved using the phone-based setup: "A third key advance in our work is the demonstration that a compact microscope based on a mobile phone and low-cost optics can accurately read the Cas13a direct detection assay, enabling ∼100 copies/μL sensitivity in 30 min of measurement and accurate diagnosis of a set of pre-extracted RNA from patient samples in 5 min of measurement time on the device."